src/test/java/api/support/fixtures/ItemsFixture.java

```diff
@@ -30,6 +30,9 @@
   private final LocationsFixture locationsFixture;
   private final RecordCreator instanceTypeRecordCreator;
   private final RecordCreator contributorNameTypeRecordCreator;
+  
+  private int index = 0;
```

Comment:
Why does the fixture need to keep this state? Could the client not do so?

Reply:
Sure.